### DISCUSSION TITLE
How many GPUs for OPENVINO and HWAccel?

### DISCUSSION THREAD

**Discussion by silentnyte** [blakeblackshear/frigate#6201] [Ask A Question]
I am using a Coffee Lake Intel Core i3-8100 CPU @ 3.60GHz with GPU.  I want to use GPU for hwaccel and the openvino detector. Should I set openvino to AUTO/GPU/CPU? If GPU how many GPUs should I allocate to the container for both hwaccel and openvino?

I currently only have 3 camera streams.

**Comment by NickM-27** [comment]
Can you clarify what you mean by how many GPUs? Are you saying your system has more than one? 

In general, the igpu should be plenty for 3 cameras hwaccel and openvino

**Reply by silentnyte** [reply]
In TrueNAS you need to allocate GPU resources to a container. I am trying to determine how many I should allocate. Frigate is the only thing I have using the GPU.
https://i.imgur.com/y4gGHEJ.png

**Reply by NickM-27** [reply]
If it's the only thing using the GPU I don't see a reason to not allocate the whole GPU
